lib/widgets/mapCardHero_widget.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:gra_terenowa/controller/mapData_controller.dart';
+import 'package:gra_terenowa/statics/colors.dart';
+import 'package:gra_terenowa/statics/constants.dart';
+import 'package:gra_terenowa/view/mapCard_screen.dart';
+
+class MapCardHero extends StatelessWidget {
```

Comment:
The list of objects is quite extensive- maybe a different layout would suit it more? I.e. 2x2 horizontal scrollable tiles? That would probably require some redesign like removing stars. Also shall the list be arranged alphabetically or according to some other logic?

Reply:
For now I was following the layout defined by our figma specs. I think it is better to wait for beta users input before entering the GUI redesign mode. Perhaps that should be our first iteration.